packages/scrollable/src/react/index.tsx

```diff
@@ -0,0 +1,206 @@
+import {
+  canUseDOM,
+  combineFns,
+  shallowCompare,
+  useResizeObserver
+} from '@pluralsight/ps-design-system-util'
+import { compose, css, media } from 'glamor'
+import React, {
+  forwardRef,
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useMemo,
+  useState,
+  useRef
+} from 'react'
+
+import stylesheet from '../css'
+
+const styles = {
+  outer: () =>
+    compose(
+      css(stylesheet['.psds-scrollable__outer']),
+      media('not print', stylesheet['.psds-scrollable__outer--screen'])
+    ),
+  inner: () => css(stylesheet['.psds-scrollable__inner']),
+  content: () => css(stylesheet['.psds-scrollable__content']),
+  handle: (grabbed: boolean) =>
+    compose(
+      css(stylesheet['.psds-scrollable__handle']),
+      grabbed && css(stylesheet['.psds-scrollable__handle--grabbed'])
+    )
+}
+
+let BLANK_IMAGE: HTMLImageElement | undefined
+if (canUseDOM()) {
+  BLANK_IMAGE = new Image(0, 0)
+  // prettier-ignore
+  BLANK_IMAGE.src = 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7';
+}
+
+function areEqualProps(
+  prevProps: Record<string, unknown>,
+  nextProps: Record<string, unknown>
+) {
+  const changed = shallowCompare(prevProps, nextProps)
+  return !changed
+}
+
+interface ScrollableProps extends React.HTMLAttributes<HTMLDivElement> {
+  renderContent?: (
+    props: renderContentProps,
+    ref: React.RefCallback<HTMLElement>
```

Comment:
should this be `React.Ref`

>     type Ref<T> = RefCallback<T> | RefObject<T> | null;

I honestly still don't know what to use for Refs

Reply:
Yes, that still works as Ref, which is the supertype, so I'll use it.

Still think a user will only be able to pass a RefCallback in practice (no RefObject via forwardRef, only LegacyRef left available), but that could just be a misunderstanding.